dev-tools/xtask/src/check_features.rs

```diff
@@ -0,0 +1,134 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Subcommand: cargo xtask check-features
+
+use anyhow::{bail, Context, Result};
+use clap::Parser;
+use std::process::Command;
+
+/// The default version of `cargo-hack` to install.
+/// We use a patch-floating version to avoid breaking the build when a new
+/// version is released (locally).
+const FLOAT_VERSION: &str = "~0.6.28";
+
+#[derive(Parser)]
+pub struct Args {
+    /// Features to exclude from the check.
+    #[clap(long)]
+    exclude_features: Option<Vec<String>>,
+    /// Depth of the feature powerset to check.
+    #[clap(long)]
+    depth: Option<usize>,
+    /// Error format passed to `cargo hack check`.
+    #[clap(long, value_name = "FMT")]
+    message_format: Option<String>,
+    /// Do not install `cargo-hack` before running the check.
+    #[clap(long, default_value_t = false)]
+    no_install: bool,
+    /// Version of `cargo-hack` to install.
+    #[clap(long)]
+    version: Option<String>,
```

Comment:
`--version` typically means the version of the program you're running. Could you replace this with `--install-version`?